### PR TITLE
Map Annotation Example

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add example on how to add map annotation

# Testing this PR

Check that the example is valid

# Related reading

https://trello.com/c/17wa1VUH/76-map-annotation-java-example2. what this PR assists, fixes, or otherwise affects

cc @bramalingam @will-moore

### CODE REVIEW

**Comment by jburel** [comment]
@joshmoore: This one (and the corresponding doc) could be a candidate for 5.4.2

File: examples/Training/java/src/training/HowToCreateMapAnnotation.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        result.add(new NamedValue("PBS", "10mM"));
+        result.add(new NamedValue("incubation", "5min"));
+        result.add(new NamedValue("temperature", "37"));
+        result.add(new NamedValue("Organism", "Homo sapiens"));
+        MapAnnotationData data = new MapAnnotationData();
+        data.setContent(result);
+        data.setDescription("Training Example");
+        //Use the following namespace if you want the annotation to be displayed
+        //in the right-hand panel
+        data.setNameSpace(MapAnnotationData.NS_CLIENT_CREATED);
```
Comment (will-moore) [inline examples/Training/java/src/training/HowToCreateMapAnnotation.java]: I guess that's the same ns as ```omero.constants.metadata.NSCLIENTMAPANNOTATION```?
The doc-string should say "...want the annotation to be *editable* in the webclient & Insight"
(All map-anns are displayed).

Comment (jburel) [reply examples/Training/java/src/training/HowToCreateMapAnnotation.java]: it is the same. I will update the doc, that's what I meant 
thanks for checking

**Comment by jburel** [comment]
Changing the base since it is a possible candidate for 5.4.2
see comment on https://trello.com/c/0q8yyo3E/71-java-training-examples-link-needs-to-be-added-to-java-bindings-page

**Comment by will-moore** [comment]
It would be nicer to have this in the WriteData.java, since we can then include it under the Write Data section of the OMERO Java docs page and users will know to find it in WriteData.java (same organisation as for Python). Thanks.

**Comment by jburel** [comment]
@will-moore done

**Comment by will-moore** [comment]
@jburel Thanks. Added to https://github.com/openmicroscopy/ome-documentation/pull/1823

**Comment by jburel** [comment]
https://ci.openmicroscopy.org/job/OMERO-DEV-merge-training/818/ is green

**Comment by bramalingam** [comment]
Tests are passing, changes look good (along with the documentation PR). Looks good to merge according to me.

**Comment by joshmoore** [comment]
:+1:

**Pull request merged**